projects/batfish/src/main/java/org/batfish/grammar/cumulus_frr/CumulusFrrConfigurationBuilder.java

```diff
@@ -1198,6 +1208,32 @@ public void exitSbb_max_med_administrative(Sbb_max_med_administrativeContext ctx
     }
   }
 
+  @Override
+  public void exitSbbl_range(Sbbl_rangeContext ctx) {
+    BgpNeighbor bgpNeighbor;
+    if (ctx.prefix() != null) {
+      Prefix listenRange = toPrefix(ctx.prefix());
+      bgpNeighbor =
+          _currentBgpVrf
+              .getNeighbors()
+              .computeIfAbsent(
+                  listenRange.toString(), (name) -> new BgpPassiveNeighbor(name, listenRange));
+    } else {
+      Prefix6 listenRange = toPrefix6(ctx.prefix6());
+      bgpNeighbor =
+          _currentBgpVrf
+              .getNeighbors()
+              .computeIfAbsent(
+                  listenRange.toString(), (name) -> new BgpPassive6Neighbor(name, listenRange));
+    }
+    bgpNeighbor.setPeerGroup(ctx.name.getText());
+  }
+
+  @Override
+  public void exitSbbl_limit(Sbbl_limitContext ctx) {
+    warn(ctx, "Batfish does not limit the number sessions for passive BGP neighbors");
```

Comment:
passive command shouldn't be bundled with bgp listen.

Reply:
Passive command is not handled in this PR. 

What are the semantics of the "listen range" command? FRR documentation (https://docs.frrouting.org/en/latest/bgp.html#clicmd-bgp-listen-range-A.B.C.D-M-X-X-X-X-M-peer-group-PGNAME) is thin, but my understanding is that the router will accept connections from the specified prefix, and it will not initiate connections. Such listening peers are "passive" in batfish terminology (some vendors call them passive, and some call them dynamic).

Is my understanding correct?

Reply:
Correct, just as a reader I was misreading the warning message :) I'd say the `passive` keyword should be replaced with `dynamic`.

Reply:
I see - happy to change. We try to use vendor-specific terminology in this part of the code. 

Do you have a pointer handy that talks about FRR terminology?

Reply:
Basically, the terminology is kinda a copy/paste from the Cisco in FRR regarding `bgp listen` command.

Reply:
https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/iproute_bgp/configuration/xe-16/irg-xe-16-book/bgp-dynamic-neighbors.html (BGP Dynamic Neighbors)

Reply:
Ok, thanks. Changed the terminology.